marketplace-ui/src/app/shared/components/footer/footer.component.spec.ts

```diff
@@ -70,4 +71,19 @@ describe('FooterComponent', () => {
     expect(getComputedStyle(logo.nativeElement).textAlign).toBe('center');
     expect(getComputedStyle(ivyPolicy.nativeElement).textAlign).toBe('center');
   });
+
+  it('should navigate to the correct URL when the link is clicked', () => {
```

Comment:
`when the link is clicked` -> `when the link icon is clicked'`

Reply:
Adjusted test according to feedback